package.json

```diff
@@ -47,5 +47,13 @@
     "test": "yarn workspaces run test",
     "test:e2e": "yarn workspace offer-up-ui test:e2e",
     "build": "yarn workspaces run build"
+  },
+  "devDependencies": {
+    "@agoric/synpress": "^3.8.1-beta.1",
+    "eslint-config-turbo": "^1.13.0",
+    "eslint-plugin-chai-friendly": "^0.7.4",
+    "eslint-plugin-cypress": "^2.15.1",
+    "eslint-plugin-testing-library": "^6.2.0",
+    "eslint-plugin-ui-testing": "^2.0.1"
```

Comment:
these are all needed by `@agoric/synpress`? maybe they should be `dependencies` so they're pulled in without this additional work and maintenance

Reply:
Yes, they're required by `@agoric/synpress`. 

And I was thinking the same about adding them as `dependencies` because it's been quite tedious to add them in every dapp I've worked with so far. Will discuss it with Fraz to confirm as he worked with the linting stuff.